Resources/Prototypes/Entities/Objects/Specific/syndicate.yml

```
@@ -83,6 +83,18 @@
     balance:
       Telecrystal: 50
 
+- type: entity
+  parent: BaseUplinkRadio
+  id: BaseUplinkRadio50TCNukeOps
+  suffix: 50 TC, NukeOps reinforcement
+  components:
+  - type: Store
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)